### ISSUE TITLE
[BUG] Cannot tell apart lit and unlit Wi-Fi icons when system Color Contrast is set to High

### ISSUE THREAD

**Issue by nyanpasu64** [Kin69/Athena#83]
## Required Information

- [x] I’m using the latest Athena version

- [ ] The device is rooted, and nothing has been removed via ADB

### Steps to Reproduce the Bug
1. In Android Settings, open Display > Color contrast.
2. Set contrast to High.
3. In Athena, tap on the Wi-Fi icon of an app to block network.

### Expected Behavior
There is a clear visual difference between apps with and without networking enabled.

### Screenshots

Normal:
<img width="720" height="1280" alt="Image" src="https://github.com/user-attachments/assets/61463166-1452-45e4-b8d7-5a25ad6b6616" />

High:
<img width="720" height="1280" alt="Image" src="https://github.com/user-attachments/assets/b5d0f8e2-e928-47cb-a064-352389d07b11" />

### Smartphone Information
- **Device:** Switch v2
- **OS:** Lineage OS 22.2 (Android 15), nx_tab
- **Athena Version:** 1.80

### Additional Context
Add any other context or details about the problem here.
